packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/store/init.ts

```diff
@@ -27,6 +27,7 @@ import {
 export function initEditorState(
   configNodes: ConfigNode[],
   uiNodes: UINode[],
+  creationMode = false,
```

Comment:
Shall we stick with `isEditing` for consistency? Using different names for the same state (`isEditing` vs. `creationMode`) could add unnecessary friction.

Reply:
Yes. Let me update this